portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +26,35 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  ArrayList<String> fun_facts = new ArrayList<String>();
```

Comment:
Java style prefers camelCase, so funFacts instead of fun_facts.

Also make sure variable is private. 
private List<String> funFacts = new ArrayList<String>();

Reply:
Done!